test-util/src/main/java/io/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -527,6 +530,15 @@ private String summarizeError(final Record<?> record) {
         .toString();
   }
 
+  private String summarizeProcessEvent(final Record<?> record) {
+    final ProcessEventRecordValue value = (ProcessEventRecordValue) record.getValue();
+    return formatId(value.getTargetElementId())
+        + "@["
+        + shortenKey(value.getScopeKey())
+        + "]"
```

Comment:
This format is slightly different from the others and doesn't match the [description](https://github.com/camunda-cloud/zeebe/blob/594e759d07a634c332c34e6304bf846cb44c88c6/test-util/src/main/java/io/zeebe/test/util/record/CompactRecordLogger.java#L167). 

We can use the existing method instead to ensure a unified format.  

```suggestion
    return summarizeElementInformation(value.getTargetElementId(), value.getScopeKey())
```